LIPs/LIP-25.md

```diff
@@ -0,0 +1,237 @@
+---
+lip: 25
+title: Extensible Governance Contract
+author: Nico Vergauwen (@kyriediculous) <nico@livepeer.org>
+type: Meta
+status: Draft
+created: 2020-06-09
+requires: n/a
```

Comment:
Let's remove this if there are no required LIPs.

Reply:
I assigned #30 as a requirement in 1afd126